connect-widget/app/src/shared/components/WidgetContainer.tsx

```diff
@@ -93,9 +102,37 @@ const WidgetContainer = ({ children }: WidgetContainerProps) => {
             <Text mr={2}>Powered by</Text>
             <Image width={150} src={require("../../assets/metriport-logo.png")}></Image>
           </Flex>
-          <Box px={8} py={6} ref={resetScroll} maxHeight={"80vh"} overflowY={"scroll"}>
+          <Box
+            px={8}
+            pt={6}
+            ref={resetScroll}
+            maxHeight={"80vh"}
+            overflowY={"scroll"}
+            onScroll={handleScroll}
+            css={{
+              "&::-webkit-scrollbar": {
+                width: "0.2rem",
+              },
+              "&::-webkit-scrollbar-thumb": {
+                background: "gray",
+                borderRadius: "0.25rem",
+              },
+            }}
+          >
             {children}
           </Box>
+          {children.props.children[0].type.name === "ConnectProviders" && !scrolled && (
+            <Box
+              position="sticky"
+              bottom={0}
+              left={0}
+              right={0}
+              display="flex"
+              justifyContent="center"
+            >
+              <ChevronDownIcon boxSize={12} />
+            </Box>
+          )}
```

Comment:
I would suggest that we make the color of the arrow like a light gray or maybe even the faded metriport purple. Also could we have the icon be at the bottom but hover over the scroll?

Reply:
Should we maybe use the same color as the overall theme, but more faded instead?
Not sure what you mean by "hover over the scroll"